docs/distributions/gentoo/installation.md

```diff
@@ -1,89 +1,56 @@
-
-5. Follow the [Gentoo Handbook](https://wiki.gentoo.org/wiki/Handbook:AMD64/Installation/Disks)
-
-    1. Instead of making a new EFI partition for Gentoo, you should instead mount `/dev/nvme0n1p1` to `/mnt/gentoo/boot/efi`
-    2. Make sure that you have deleted the partition you made ahead of time, or else you'll lose important data
-
-6. Continue through the Handbook until chapter "Configuring the kernel"
-
-    1. In order to have the Linux Kernel support your device the most, you'll have to build from a modified Linux Kernel source tree.
-    2. Git clone [the t2linux kernel source tree](https://github.com/t2linux/kernel) to `/usr/src`
-        * Make sure to git checkout the version you want. For example, if you want kernel version v5.16.17, you would checkout tag `t2-v5.16.17`.
-    3. Run these commands to symlink the kernel to `/usr/src/linux`:
+    1. In order to build support for T2 MacBooks in the kernel, you'll have to apply a patchset and configure the kernel manually.
+    2. Run these commands to enable the patchset and install the kernel sources:
 
     ```bash
-    eselect kernel list # there should be one option that shows up
-    eselect kernel set 1
-    ls -l /usr/src/linux # check if the symlink succeded
+    git clone https://github.com/t2linux/linux-t2-patches /etc/portage/patches/gentoo-sources
+    rm -rf /etc/portage/package.accept_keywords
+    echo "sys-kernel/gentoo-sources ~amd64" >> /etc/portage/package.accept_keywords
+    emerge --ask sys-kernel/gentoo-sources
     ```
 
-    1. Run these commands to grab apple-bce and apple-ibridge:
+    1. Follow [this section](https://wiki.gentoo.org/wiki/Handbook:AMD64/Installation/Kernel#Manual_configuration) of the Gentoo handbook.
+    2. Make sure the kernel config options `CONFIG_HID_APPLE_IBRIDGE`, `CONFIG_HID_APPLE_TOUCHBAR`, `CONFIG_HID_APPLE_MAGIC_BACKLIGHT`, `CONFIG_BT_HCIBCM4377`, `CONFIG_BRCMFMAC`, and `CONFIG_APPLE_BCE` are all set to M before compiling. Make sure the options `BRCMFMAC_PCIE` and `CONFIG_IRQ_REMAP` are set to Y before building the kernel.
+    3. Build the kernel using the instructions found in the [https://wiki.gentoo.org/wiki/Handbook:AMD64/Installation/Kernel#Compiling_and_installing](Handbook).
 
-    ```bash
-    git clone https://github.com/t2linux/apple-bce-drv /usr/src/apple-bce
-    git clone https://github.com/t2linux/apple-ib-drv /usr/src/apple-ibridge
-    for i in apple-bce apple-ibridge; do 
-    mkdir /usr/src/linux/drivers/staging/$i 
-    cp -r /usr/src/$i/* /usr/src/linux/drivers/staging/$i/
-    done
-    ```
-
-    1. Run these commands in order to include apple-bce and apple-ibridge into the kernel source tree:
-
-    ```bash
-    git clone https://github.com/Redecorating/mbp-16.1-linux-wifi /linux-patches
-    cd /usr/src/linux
-    for i in /linux-patches/*.patch; do
-    echo $i
-    patch -Np1 < $i
-    done
-    ```
+4. Build an initramfs using `dracut`.
```

Comment:
Instead of having this as a separate step, you should inform the user in the previous step to follow the "Building the initramfs" section, along with the extra instruction to include the `apple-bce` driver.